data.go

```diff
@@ -77,7 +77,12 @@ func (dbc *dbManager) getDb() apid.DB {
 }
 
 func (dbc *dbManager) initDb() error {
-	_, err := dbc.getDb().Exec(`
+	tx, err := dbc.getDb().Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback()
```

Comment:
Ideally, If tx.Rollback() were to fail, the DB is in a bad state that we shouldn't continue and mess the DB further. Not a big deal, but worth adding later.